docs/versioned_docs/version-2.11.0/user-guides/jivaguide.md

```diff
@@ -0,0 +1,958 @@
+
+You can specify the *ReplicaResourceLimits* to restrict the memory usage of replica pod within the given limit using the *value* for *ReplicaResourceLimits*.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: ReplicaResourceLimits
+        value: |-
+            memory: 2Gi
+    openebs.io/cas-type: jiva
+provisioner: openebs.io/provisioner-iscsi
+```
+
+#### Target Affinity Policy
+
+The Stateful workloads access the OpenEBS storage volume by connecting to the Volume Target Pod. This policy can be used to co-locate volume target pod on the same node as workload.
+
+- This feature makes use of the Kubernetes Pod Affinity feature that is dependent on the Pod labels. User will need to add the following label to both Application and PVC.
+
+  ```
+  labels:
+    openebs.io/target-affinity: <application-unique-label>
+  ```
+
+- You can specify the Target Affinity in both application and OpenEBS PVC using the following way. 
+
+  The following is a snippet of an application deployment YAML spec for implementing target affinity. 
+
+  ```
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: fio-jiva
+    labels:
+      name: fio-jiva
+      openebs.io/target-affinity: fio-jiva
+  ```
+
+  For OpenEBS PVC, it will be similar to the following.
+
+  ```
+  kind: PersistentVolumeClaim
+  apiVersion: v1
+  metadata:
+    name: fio-jiva-claim
+    labels:
+      openebs.io/target-affinity: fio-jiva
+  ```
+
+**Note**: *This feature works only for cases where there is a single application pod instance associated to a PVC.  Example YAML spec for application deployment can be get from [here](https://raw.githubusercontent.com/openebs/openebs/master/k8s/demo/fio/demo-fio-jiva-taa.yaml). In the case of STS, this feature is supported only for single replica StatefulSet.*
+
+#### OpenEBS Namespace Policy for Jiva Pods
+
+This StorageClass Policy is for deploying the Jiva pods in OpenEBS Namespace. By default, the value is `false`, so Jiva Pods will deploy in PVC namespace. The following are the main requirement of running Jiva pods in OpenEBS namespace.
+
+* With default value, granting additional privileges to Jiva pods to access hostpath might involve granting privileges to the entire namespace of PVC. With enabling this value as`true` , Jiva pods will get additional privileges to access hostpath in OpenEBS namespace. 
+
+* To avoid duplicate Jiva Pod creation during the restoration using Velero.
+
+  The following is a snippet of an StorageClass YAML spec for running Jiva pods in `openebs` namespace.
+
+  ```
+  apiVersion: storage.k8s.io/v1
+  kind: StorageClass
+  metadata:
+    name: jiva-pods-in-openebs-ns
+    annotations:
+      openebs.io/cas-type: jiva
+      cas.openebs.io/config: |
+        - name: DeployInOpenEBSNamespace
+          enabled: "true"
+  provisioner: openebs.io/provisioner-iscsi
+  ```
+
+## See Also:
+
+[Understanding Jiva](/concepts/jiva)
```

Comment:
will this link work? (shouldn't we prefixed it with `/docs`?)

Reply:
we dont have to use `/docs` on links, it will work